Geant4.10 polarimeter

sjdkay opened the pull request:
Implemented PID-III and polarimeter construction with options.

Also added ability to modify Z position of various components.

Added in a smaller hydrogen target (to represent the new target nose).

Also have folder (npol) which has (currently not implemented) stuff to deal with polarised particles.

sjdkay committed:
Added new macros

sjdkay committed:
Updated macros

sjdkay committed:
Added in Polarimeter to Geant simulation

sjdkay committed:
New macros for single and double pi0 production cases, trying to add geometry test to main macro

sjdkay committed:
Created new skeleton polarimeter detector files, added a third construction option for the PID to make the new mini PID III

sjdkay committed:
Added in offset parameter for MWPC, works on main chamber but doesn't look like support stucture is moving so still needs work!

sjdkay committed:
Added another option for PID III which is shorter in length, added descriptions to setup macro

sjdkay committed:
Modified A2DetConstruction.cc

sjdkay committed:
Added ability to offset target cell along Z axis (For cryo target only currently!)

sjdkay committed:
Fixed some problems with cryo target Z position adjustment, only ever needed to change the mother volume part!

sjdkay committed:
Changed radius of cryo target, will update this to be a new option in future (i.e. will be able to select old 4.1cm or new 3.1cm radius target

sjdkay committed:
Added in updated PID option, also now have the option to select which of the two cryo target versions to use, the new smaller one or the older larger one. Next step is to add new lightguides for PID option 4. Also target selection workaround is slightly messy

sjdkay committed:
Polarimeter now added as a detector, can select in config which version (phase I or II currently, planning to add some Phase II options soon) you want. Need to actually check that this builds the detector correctly, it APPEARS to do so but it may not be in the right place (can't viualise currently)

sjdkay committed:
Changes to polarimeter detector class, should match existing way of making polarimeter in detector construction now

sjdkay committed:
Uncommented Phase I cap construction, should work now?

sjdkay committed:
Clarified some outputs to screen (e.g. which Z position belongs to which part)

sjdkay committed:
Added in new construction option for Phase II polarimeter and also fixed error where Phase II polarimeter was creating a cap

sjdkay committed:
Delete .nfs000000000b3c57ed00000061

sjdkay committed:
Fixed positioning of new polarimeter and added in z position adjustment functionality

sjdkay committed:
Added new angled light guides for new PID construction option, alignment may not be completely correct (or displacement of PID)

sjdkay committed:
Removed old Polarisation stuff in preparation for adding in new polarised physics classes

sjdkay committed:
Removed old Polarisation stuff in preparation for adding in new polarised physics classes

sjdkay committed:
Updated larger version of new PID to reflect current design, this includes adding in bent light guides and has the correct shifts required for the polarimeter and PID. Also reduced size of polarimeter/PID to reflect current designs

sjdkay committed:
Updated in accordance with patch from Lorenzo, patch fixes vertex creation if created outside of target

sjdkay committed:
Added in slanted end of PID scintillators for two new options, need to run simulation later

sjdkay committed:
Fixed errors for PID 2 (Al ring added back in) and PID3v1 (Mother volume outer radius too small), updated output text in detector messenger

sjdkay committed:
Added in smaller PID variants for use with new polarimeters with cap, setup macros updated with new descriptions and instructions

sjdkay committed:
Minor changes to PID/Pol source files, comments clarified in some cases and work to be done added to top as a comment

sjdkay committed:
Now have option for P2 polarimeter to build with or without cap, still need to update dimensions to current drawings

sjdkay committed:
Added working input for PID end piece construction selection, still need to modify actual end pieces to fit in with those to be used in reality however

sjdkay committed:
Added in proposed support for new polarimeter, fixed issue with polarimeter end cap positioning

sjdkay committed:
Fixed error in material for polarimeter support pipes

sjdkay committed:
Minor changes

sjdkay committed:
Updated PID ends to look more like those in reality

sjdkay committed:
Fixed MWPC Z position adjustment

sjdkay committed:
Removed redundant PID options, made polarimeter cap correct. Added backup polarimeter tube as an option

lorenzozana commented:
You are tracking in your changes also the build directory . You shouldn't since that one is removed and replaced at each installation. Also you should not track root files that are not needed.

Lorenzo 

Sent from my iPhone

> On Aug 2, 2016, at 3:41 PM, Stephen Kay notifications@github.com wrote:
> 
> Implemented PID-III and polarimeter construction with options.
> 
> Also added ability to modify Z position of various components.
> 
> Added in a smaller hydrogen target (to represent the new target nose).
> 
> Also have folder (npol) which has (currently not implemented) stuff to deal with polarised particles.
> 
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/A2-Collaboration/a2geant/pull/2
> 
> Commit Summary
> 
> Added new macros
> Updated macros
> Added in Polarimeter to Geant simulation
> New macros for single and double pi0 production cases, trying to add geometry test to main macro
> Created new skeleton polarimeter detector files, added a third construction option for the PID to make the new mini PID III
> Added in offset parameter for MWPC, works on main chamber but doesn't look like support stucture is moving so still needs work!
> Added another option for PID III which is shorter in length, added descriptions to setup macro
> Modified A2DetConstruction.cc
> Added ability to offset target cell along Z axis (For cryo target only currently!)
> Fixed some problems with cryo target Z position adjustment, only ever needed to change the mother volume part!
> Changed radius of cryo target, will update this to be a new option in future (i.e. will be able to select old 4.1cm or new 3.1cm radius target
> Added in updated PID option, also now have the option to select which of the two cryo target versions to use, the new smaller one or the older larger one. Next step is to add new lightguides for PID option 4. Also target selection workaround is slightly messy
> Polarimeter now added as a detector, can select in config which version (phase I or II currently, planning to add some Phase II options soon) you want. Need to actually check that this builds the detector correctly, it APPEARS to do so but it may not be in the right place (can't viualise currently)
> Changes to polarimeter detector class, should match existing way of making polarimeter in detector construction now
> Uncommented Phase I cap construction, should work now?
> Clarified some outputs to screen (e.g. which Z position belongs to which part)
> Added in new construction option for Phase II polarimeter and also fixed error where Phase II polarimeter was creating a cap
> Delete .nfs000000000b3c57ed00000061
> Fixed positioning of new polarimeter and added in z position adjustment functionality
> Added new angled light guides for new PID construction option, alignment may not be completely correct (or displacement of PID)
> Removed old Polarisation stuff in preparation for adding in new polarised physics classes
> Removed old Polarisation stuff in preparation for adding in new polarised physics classes
> Updated larger version of new PID to reflect current design, this includes adding in bent light guides and has the correct shifts required for the polarimeter and PID. Also reduced size of polarimeter/PID to reflect current designs
> Updated in accordance with patch from Lorenzo, patch fixes vertex creation if created outside of target
> Added in slanted end of PID scintillators for two new options, need to run simulation later
> Fixed errors for PID 2 (Al ring added back in) and PID3v1 (Mother volume outer radius too small), updated output text in detector messenger
> Added in smaller PID variants for use with new polarimeters with cap, setup macros updated with new descriptions and instructions
> Minor changes to PID/Pol source files, comments clarified in some cases and work to be done added to top as a comment
> Now have option for P2 polarimeter to build with or without cap, still need to update dimensions to current drawings
> Added working input for PID end piece construction selection, still need to modify actual end pieces to fit in with those to be used in reality however
> Added in proposed support for new polarimeter, fixed issue with polarimeter end cap positioning
> Fixed error in material for polarimeter support pipes
> Minor changes
> Updated PID ends to look more like those in reality
> Fixed MWPC Z position adjustment
> Removed redundant PID options, made polarimeter cap correct. Added backup polarimeter tube as an option
> File Changes
> 
> M include/A2CBOutput.hh (9)
> M include/A2CryoTarget.hh (4)
> M include/A2DetMWPC.hh (5)
> M include/A2DetPID.hh (32)
> A include/A2DetPol.hh (63)
> M include/A2DetectorConstruction.hh (60)
> M include/A2DetectorMessenger.hh (28)
> M include/A2PhysicsList.hh (27)
> M include/A2PhysicsListMessenger.hh (15)
> M include/A2PolarizedTarget.hh (8)
> M include/A2SolidTarget.hh (4)  
> M include/A2Target.hh (6)
> M macros/DetectorSetup.mac (25)
> A macros/DetectorSetup_Empty.mac (43)
> A macros/EdGen.mac (26)
> A macros/EdGen_2Pi0.mac (25)
> A macros/EdGen_Beam.mac (26)
> A macros/EdGen_Neutrons.mac (24)
> A macros/EdGen_Pi0.mac (24)
> A macros/EdGen_Protons.mac (24)
> A macros/ListPhysics.mac (5)
> M macros/doBeamOnTarget.mac (31)
> M macros/vis.mac (39)
> A npol/CMakeLists.txt (79)
> A npol/G4AnalysingPower.lyx (559)
> A npol/G4AnalysingPower.pdf (0)
> A npol/INSTALL (9)
> A npol/Reconstruct.C (98)
> A npol/Reconstruct_C.d (85)
> A npol/bugs.txt (7)
> A npol/build/CMakeCache.txt (331)
> A npol/build/CMakeFiles/2.8.12.2/CMakeCCompiler.cmake (56)
> A npol/build/CMakeFiles/2.8.12.2/CMakeCXXCompiler.cmake (57)
> A npol/build/CMakeFiles/2.8.12.2/CMakeDetermineCompilerABI_C.bin (0)
> A npol/build/CMakeFiles/2.8.12.2/CMakeDetermineCompilerABI_CXX.bin (0)
> A npol/build/CMakeFiles/2.8.12.2/CMakeSystem.cmake (15)
> A npol/build/CMakeFiles/2.8.12.2/CompilerIdC/CMakeCCompilerId.c (389)
> A npol/build/CMakeFiles/2.8.12.2/CompilerIdC/a.out (0)
> A npol/build/CMakeFiles/2.8.12.2/CompilerIdCXX/CMakeCXXCompilerId.cpp (377)
> A npol/build/CMakeFiles/2.8.12.2/CompilerIdCXX/a.out (0)
> A npol/build/CMakeFiles/CMakeDirectoryInformation.cmake (16)
> A npol/build/CMakeFiles/CMakeOutput.log (247)
> A npol/build/CMakeFiles/Makefile.cmake (51)
> A npol/build/CMakeFiles/Makefile2 (102)
> A npol/build/CMakeFiles/TargetDirectories.txt (1)
> A npol/build/CMakeFiles/cmake.check_cache (1)
> A npol/build/CMakeFiles/npol.dir/CXX.includecache (2432)
> A npol/build/CMakeFiles/npol.dir/DependInfo.cmake (201)
> A npol/build/CMakeFiles/npol.dir/build.make (584)
> A npol/build/CMakeFiles/npol.dir/cmake_clean.cmake (26)
> A npol/build/CMakeFiles/npol.dir/depend.internal (6384)
> A npol/build/CMakeFiles/npol.dir/depend.make (6384)
> A npol/build/CMakeFiles/npol.dir/flags.make (8)
> A npol/build/CMakeFiles/npol.dir/link.txt (1)
> A npol/build/CMakeFiles/npol.dir/progress.make (18)
> A npol/build/CMakeFiles/progress.marks (1)
> A npol/build/Makefile (643)
> A npol/build/QuasiFreeApow.root (0)
> A npol/build/QuasiFreeApow.root.old (0)
> A npol/build/cmake_install.cmake (62)
> A npol/build/include/CMakeG4Env.hh (15)
> A npol/build/npol (0)
> A npol/build/polqf.mac (34)
> A npol/build/testhist.root (0)
> A npol/cmake/Modules/FindGeant4.cmake (63)
> A npol/cmake/Modules/FindROOT.cmake (161)
> A npol/cmake/Modules/FindROOTHelpers.cmake (43)
> A npol/cmake/g4env.sh (3)
> A npol/cmake/settings.cmake (5)
> A npol/include/PolHadronElasticPhysics.hh (95)
> A npol/include/PolHadronElasticProcess.hh (83)
> A npol/include/PolHadronInelasticPhysics.hh (56)
> A npol/include/PolHadronInelasticProcess.hh (33)
> A npol/include/PolHadronQuasiFreeProcess.hh (32)
> A npol/include/PolHadronicProcess.hh (119)
> A npol/include/PolNPRotate.hh (88)
> A npol/include/PolNucleonRotate.hh (120)
> A npol/include/PolPhysicsList.hh (92)
> A npol/include/PolPhysicsListMessenger.hh (67)
> A npol/include/npolDetCon.hh (51)
> A npol/include/npolEventAction.hh (64)
> A npol/include/npolHit.hh (105)  
> A npol/include/npolPrimGen.hh     (36)
> A npol/include/npolSD.hh (67)
> A npol/include/npolSteppingAction.hh (28)
> A npol/makePatch10.01.p02.csh (20)
> A npol/npol.cc (121)
> A npol/patch_geant4.10.01.p02.JRMA.tar.gz (0)
> A npol/plots/Neutron400_Scint.gif (0)
> A npol/plots/Neutron400_lH.gif (0)
> A npol/plots/NeutronMc.gif (0)
> A npol/plots/NeutronMc_ThS10.gif (0)
> A npol/plots/NeutronMc_ThS20.gif (0)
> A npol/plots/NeutronMc_ThS30.gif (0)
> A npol/plots/NeutronMc_ThS40.gif (0)
> A npol/plots/NeutronMcchEx_ThS10.gif (0)
> A npol/plots/NeutronMcchEx_ThS20.gif (0)
> A npol/plots/NeutronMcchEx_ThS30.gif (0)
> A npol/plots/NeutronMcchEx_ThS40.gif (0)
> A npol/plots/NeutronQF.gif (0)
> A npol/plots/NeutronQF_ThS10.gif (0)
> A npol/plots/NeutronQF_ThS20.gif (0)
> A npol/plots/NeutronQF_ThS30.gif (0)
> A npol/plots/NeutronQF_ThS40.gif (0)
> A npol/plots/NeutronQFchEx.gif (0)
> A npol/plots/NeutronQFchEx_ThS10.gif (0)
> A npol/plots/NeutronQFchEx_ThS20.gif (0)
> A npol/plots/NeutronQFchEx_ThS30.gif (0)
> A npol/plots/NeutronQFchEx_ThS40.gif (0)
> A npol/plots/Proton_lH.gif (0)
> A npol/src/PolHadronElasticPhysics.cc (264)
> A npol/src/PolHadronElasticProcess.cc (307)
> A npol/src/PolHadronInelasticPhysics.cc (90)
> A npol/src/PolHadronInelasticProcess.cc (17)
> A npol/src/PolHadronQuasiFreeProcess.cc (189)
> A npol/src/PolHadronicProcess.cc (373)
> A npol/src/PolNPRotate.cc (38)  
> A npol/src/PolNucleonRotate.cc (309)
> A npol/src/PolPhysicsList.cc (447)
> A npol/src/PolPhysicsListMessenger.cc (233)
> A npol/src/npolDetCon.cc (129)  
> A npol/src/npolEventAction.cc (173)
> A npol/src/npolHit.cc (85)
> A npol/src/npolPrimGen.cc (68)  
> A npol/src/npolSD.cc (161)
> A npol/src/npolSteppingAction.cc     (74)
> A npol/test.root (0)
> A npol/vis.mac (57)
> M src/A2CBOutput.cc (3)
> M src/A2CryoTarget.cc (296)
> M src/A2DetMWPC.cc (42)
> M src/A2DetPID.cc (331)
> A src/A2DetPol.cc (215)
> M src/A2DetectorConstruction.cc     (268)
> M src/A2DetectorMessenger.cc (107)
> M src/A2PhysicsList.cc (76)
> M src/A2PolarizedTarget.cc (48)
> M src/A2PrimaryGeneratorAction.cc (29)
> M src/A2SolidTarget.cc (27)
> Patch Links:
> 
> https://github.com/A2-Collaboration/a2geant/pull/2.patch
> https://github.com/A2-Collaboration/a2geant/pull/2.diff
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or mute the thread.

sjdkay commented:
Ok I'll ditch those and put the request in again.

Thanks, Stephen

On 2 August 2016 at 15:22, Lorenzo Zana notifications@github.com wrote:

> You are tracking in your changes also the build directory . You shouldn't
> since that one is removed and replaced at each installation. Also you
> should not track root files that are not needed.
> 
> Lorenzo
> 
> Sent from my iPhone
> 
> > On Aug 2, 2016, at 3:41 PM, Stephen Kay notifications@github.com
> > wrote:
> > 
> > Implemented PID-III and polarimeter construction with options.
> > 
> > Also added ability to modify Z position of various components.
> > 
> > Added in a smaller hydrogen target (to represent the new target nose).
> > 
> > Also have folder (npol) which has (currently not implemented) stuff to
> > deal with polarised particles.
> > 
> > You can view, comment on, or merge this pull request online at:
> > 
> > https://github.com/A2-Collaboration/a2geant/pull/2
> > 
> > Commit Summary
> > 
> > Added new macros
> > Updated macros
> > Added in Polarimeter to Geant simulation
> > New macros for single and double pi0 production cases, trying to add
> > geometry test to main macro
> > Created new skeleton polarimeter detector files, added a third
> > construction option for the PID to make the new mini PID III
> > Added in offset parameter for MWPC, works on main chamber but doesn't
> > look like support stucture is moving so still needs work!
> > Added another option for PID III which is shorter in length, added
> > descriptions to setup macro
> > Modified A2DetConstruction.cc
> > Added ability to offset target cell along Z axis (For cryo target only
> > currently!)
> > Fixed some problems with cryo target Z position adjustment, only ever
> > needed to change the mother volume part!
> > Changed radius of cryo target, will update this to be a new option in
> > future (i.e. will be able to select old 4.1cm or new 3.1cm radius target
> > Added in updated PID option, also now have the option to select which of
> > the two cryo target versions to use, the new smaller one or the older
> > larger one. Next step is to add new lightguides for PID option 4. Also
> > target selection workaround is slightly messy
> > Polarimeter now added as a detector, can select in config which version
> > (phase I or II currently, planning to add some Phase II options soon) you
> > want. Need to actually check that this builds the detector correctly, it
> > APPEARS to do so but it may not be in the right place (can't viualise
> > currently)
> > Changes to polarimeter detector class, should match existing way of
> > making polarimeter in detector construction now
> > Uncommented Phase I cap construction, should work now?
> > Clarified some outputs to screen (e.g. which Z position belongs to which
> > part)
> > Added in new construction option for Phase II polarimeter and also fixed
> > error where Phase II polarimeter was creating a cap
> > Delete .nfs000000000b3c57ed00000061
> > Fixed positioning of new polarimeter and added in z position adjustment
> > functionality
> > Added new angled light guides for new PID construction option, alignment
> > may not be completely correct (or displacement of PID)
> > Removed old Polarisation stuff in preparation for adding in new
> > polarised physics classes
> > Removed old Polarisation stuff in preparation for adding in new
> > polarised physics classes
> > Updated larger version of new PID to reflect current design, this
> > includes adding in bent light guides and has the correct shifts required
> > for the polarimeter and PID. Also reduced size of polarimeter/PID to
> > reflect current designs
> > Updated in accordance with patch from Lorenzo, patch fixes vertex
> > creation if created outside of target
> > Added in slanted end of PID scintillators for two new options, need to
> > run simulation later
> > Fixed errors for PID 2 (Al ring added back in) and PID3v1 (Mother volume
> > outer radius too small), updated output text in detector messenger
> > Added in smaller PID variants for use with new polarimeters with cap,
> > setup macros updated with new descriptions and instructions
> > Minor changes to PID/Pol source files, comments clarified in some cases
> > and work to be done added to top as a comment
> > Now have option for P2 polarimeter to build with or without cap, still
> > need to update dimensions to current drawings
> > Added working input for PID end piece construction selection, still need
> > to modify actual end pieces to fit in with those to be used in reality
> > however
> > Added in proposed support for new polarimeter, fixed issue with
> > polarimeter end cap positioning
> > Fixed error in material for polarimeter support pipes
> > Minor changes
> > Updated PID ends to look more like those in reality
> > Fixed MWPC Z position adjustment
> > Removed redundant PID options, made polarimeter cap correct. Added
> > backup polarimeter tube as an option
> > File Changes
> > 
> > M include/A2CBOutput.hh (9)
> > M include/A2CryoTarget.hh (4)
> > M include/A2DetMWPC.hh (5)
> > M include/A2DetPID.hh (32)
> > A include/A2DetPol.hh (63)
> > M include/A2DetectorConstruction.hh (60)
> > M include/A2DetectorMessenger.hh (28)
> > M include/A2PhysicsList.hh (27)
> > M include/A2PhysicsListMessenger.hh (15)
> > M include/A2PolarizedTarget.hh (8)
> > M include/A2SolidTarget.hh (4)
> > M include/A2Target.hh (6)
> > M macros/DetectorSetup.mac (25)
> > A macros/DetectorSetup_Empty.mac (43)
> > A macros/EdGen.mac (26)
> > A macros/EdGen_2Pi0.mac (25)
> > A macros/EdGen_Beam.mac (26)
> > A macros/EdGen_Neutrons.mac (24)
> > A macros/EdGen_Pi0.mac (24)
> > A macros/EdGen_Protons.mac (24)
> > A macros/ListPhysics.mac (5)
> > M macros/doBeamOnTarget.mac (31)
> > M macros/vis.mac (39)
> > A npol/CMakeLists.txt (79)
> > A npol/G4AnalysingPower.lyx (559)
> > A npol/G4AnalysingPower.pdf (0)
> > A npol/INSTALL (9)
> > A npol/Reconstruct.C (98)
> > A npol/Reconstruct_C.d (85)
> > A npol/bugs.txt (7)
> > A npol/build/CMakeCache.txt (331)
> > A npol/build/CMakeFiles/2.8.12.2/CMakeCCompiler.cmake (56)
> > A npol/build/CMakeFiles/2.8.12.2/CMakeCXXCompiler.cmake (57)
> > A npol/build/CMakeFiles/2.8.12.2/CMakeDetermineCompilerABI_C.bin (0)
> > A npol/build/CMakeFiles/2.8.12.2/CMakeDetermineCompilerABI_CXX.bin (0)
> > A npol/build/CMakeFiles/2.8.12.2/CMakeSystem.cmake (15)
> > A npol/build/CMakeFiles/2.8.12.2/CompilerIdC/CMakeCCompilerId.c (389)
> > A npol/build/CMakeFiles/2.8.12.2/CompilerIdC/a.out (0)
> > A npol/build/CMakeFiles/2.8.12.2/CompilerIdCXX/CMakeCXXCompilerId.cpp
> > (377)
> > A npol/build/CMakeFiles/2.8.12.2/CompilerIdCXX/a.out (0)
> > A npol/build/CMakeFiles/CMakeDirectoryInformation.cmake (16)
> > A npol/build/CMakeFiles/CMakeOutput.log (247)
> > A npol/build/CMakeFiles/Makefile.cmake (51)
> > A npol/build/CMakeFiles/Makefile2 (102)
> > A npol/build/CMakeFiles/TargetDirectories.txt (1)
> > A npol/build/CMakeFiles/cmake.check_cache (1)
> > A npol/build/CMakeFiles/npol.dir/CXX.includecache (2432)
> > A npol/build/CMakeFiles/npol.dir/DependInfo.cmake (201)
> > A npol/build/CMakeFiles/npol.dir/build.make (584)
> > A npol/build/CMakeFiles/npol.dir/cmake_clean.cmake (26)
> > A npol/build/CMakeFiles/npol.dir/depend.internal (6384)
> > A npol/build/CMakeFiles/npol.dir/depend.make (6384)
> > A npol/build/CMakeFiles/npol.dir/flags.make (8)
> > A npol/build/CMakeFiles/npol.dir/link.txt (1)
> > A npol/build/CMakeFiles/npol.dir/progress.make (18)
> > A npol/build/CMakeFiles/progress.marks (1)
> > A npol/build/Makefile (643)
> > A npol/build/QuasiFreeApow.root (0)
> > A npol/build/QuasiFreeApow.root.old (0)
> > A npol/build/cmake_install.cmake (62)
> > A npol/build/include/CMakeG4Env.hh (15)
> > A npol/build/npol (0)
> > A npol/build/polqf.mac (34)
> > A npol/build/testhist.root (0)
> > A npol/cmake/Modules/FindGeant4.cmake (63)
> > A npol/cmake/Modules/FindROOT.cmake (161)
> > A npol/cmake/Modules/FindROOTHelpers.cmake (43)
> > A npol/cmake/g4env.sh (3)
> > A npol/cmake/settings.cmake (5)
> > A npol/include/PolHadronElasticPhysics.hh (95)
> > A npol/include/PolHadronElasticProcess.hh (83)
> > A npol/include/PolHadronInelasticPhysics.hh (56)
> > A npol/include/PolHadronInelasticProcess.hh (33)
> > A npol/include/PolHadronQuasiFreeProcess.hh (32)
> > A npol/include/PolHadronicProcess.hh (119)
> > A npol/include/PolNPRotate.hh (88)
> > A npol/include/PolNucleonRotate.hh (120)
> > A npol/include/PolPhysicsList.hh (92)
> > A npol/include/PolPhysicsListMessenger.hh (67)
> > A npol/include/npolDetCon.hh (51)
> > A npol/include/npolEventAction.hh (64)
> > A npol/include/npolHit.hh (105)
> > A npol/include/npolPrimGen.hh (36)
> > A npol/include/npolSD.hh (67)
> > A npol/include/npolSteppingAction.hh (28)
> > A npol/makePatch10.01.p02.csh (20)
> > A npol/npol.cc (121)
> > A npol/patch_geant4.10.01.p02.JRMA.tar.gz (0)
> > A npol/plots/Neutron400_Scint.gif (0)
> > A npol/plots/Neutron400_lH.gif (0)
> > A npol/plots/NeutronMc.gif (0)
> > A npol/plots/NeutronMc_ThS10.gif (0)
> > A npol/plots/NeutronMc_ThS20.gif (0)
> > A npol/plots/NeutronMc_ThS30.gif (0)
> > A npol/plots/NeutronMc_ThS40.gif (0)
> > A npol/plots/NeutronMcchEx_ThS10.gif (0)
> > A npol/plots/NeutronMcchEx_ThS20.gif (0)
> > A npol/plots/NeutronMcchEx_ThS30.gif (0)
> > A npol/plots/NeutronMcchEx_ThS40.gif (0)
> > A npol/plots/NeutronQF.gif (0)
> > A npol/plots/NeutronQF_ThS10.gif (0)
> > A npol/plots/NeutronQF_ThS20.gif (0)
> > A npol/plots/NeutronQF_ThS30.gif (0)
> > A npol/plots/NeutronQF_ThS40.gif (0)
> > A npol/plots/NeutronQFchEx.gif (0)
> > A npol/plots/NeutronQFchEx_ThS10.gif (0)
> > A npol/plots/NeutronQFchEx_ThS20.gif (0)
> > A npol/plots/NeutronQFchEx_ThS30.gif (0)
> > A npol/plots/NeutronQFchEx_ThS40.gif (0)
> > A npol/plots/Proton_lH.gif (0)
> > A npol/src/PolHadronElasticPhysics.cc (264)
> > A npol/src/PolHadronElasticProcess.cc (307)
> > A npol/src/PolHadronInelasticPhysics.cc (90)
> > A npol/src/PolHadronInelasticProcess.cc (17)
> > A npol/src/PolHadronQuasiFreeProcess.cc (189)
> > A npol/src/PolHadronicProcess.cc (373)
> > A npol/src/PolNPRotate.cc (38)
> > A npol/src/PolNucleonRotate.cc (309)
> > A npol/src/PolPhysicsList.cc (447)
> > A npol/src/PolPhysicsListMessenger.cc (233)
> > A npol/src/npolDetCon.cc (129)
> > A npol/src/npolEventAction.cc (173)
> > A npol/src/npolHit.cc (85)
> > A npol/src/npolPrimGen.cc (68)
> > A npol/src/npolSD.cc (161)
> > A npol/src/npolSteppingAction.cc (74)
> > A npol/test.root (0)
> > A npol/vis.mac (57)
> > M src/A2CBOutput.cc (3)
> > M src/A2CryoTarget.cc (296)
> > M src/A2DetMWPC.cc (42)
> > M src/A2DetPID.cc (331)
> > A src/A2DetPol.cc (215)
> > M src/A2DetectorConstruction.cc (268)
> > M src/A2DetectorMessenger.cc (107)
> > M src/A2PhysicsList.cc (76)
> > M src/A2PolarizedTarget.cc (48)
> > M src/A2PrimaryGeneratorAction.cc (29)
> > M src/A2SolidTarget.cc (27)
> > Patch Links:
> > 
> > https://github.com/A2-Collaboration/a2geant/pull/2.patch
> > https://github.com/A2-Collaboration/a2geant/pull/2.diff
> > —
> > You are receiving this because you are subscribed to this thread.
> > Reply to this email directly, view it on GitHub, or mute the thread.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> https://github.com/A2-Collaboration/a2geant/pull/2#issuecomment-236919596,
> or mute the thread
> https://github.com/notifications/unsubscribe-auth/AJTAHA6fOdYdB3iJ9sEFfZyTSLFwlU7Tks5qb1KIgaJpZM4JanmP
> .

sjdkay committed:
Removed npol folder from branch until fully implemented

sjdkay committed:
Shortened length of support pipe for polarimeter to match reality
